### PR TITLE
Add lns support

### PR DESCRIPTION
**Pull request by mainnet-pat**
This renders LNS name and avatar when connected to a wallet address with an LNS registered
image
Also in expert mode swap recipient can be set to an LNS name
image

### CODE REVIEW

**Comment by bchpete** [comment]
LNS Support has already been added through another PR, thanks though ✌

**Comment by emberswap** [comment]
LNS Support has already been added through another PR, thanks though ✌

File: src/components/Web3Status/index.tsx [hunk 1]
```diff
@@ -124,7 +130,7 @@ function Web3StatusInner() {
       return (
         <div
           id="web3-status-connected"
-          className="flex items-center px-3 py-2 text-bold rounded-lg bg-transparent text-sm"
+          className="flex items-center px-3 py-2 text-bold rounded-lg text-sm bg-dark-800 shadow-sm hover:bg-dark-700"
```
Comment (mainnet-pat) [inline src/components/Web3Status/index.tsx:133]: look at this custom styling which makes it more similar to the neighbouring BCH and Ember flairs.

File: src/components/Web3Status/index.tsx [hunk 2]
```diff
@@ -135,8 +141,19 @@ function Web3StatusInner() {
               <Loader stroke="white" />
             </div>
           ) : (
-            <div className="mr-2">{ENSName || shortenAddress(account)}</div>
-          )}
+          <div className="flex items-center gap-2">
+            <Image
```
Comment (mainnet-pat) [inline src/components/Web3Status/index.tsx:145]: Also putting image in front makes it nice

**Pull request closed without merging**